Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -309,9 +321,10 @@ - (void)drawerPositionDidChangeWithDrawer:(PulleyViewController *)drawer bottomS
 
     BOOL isDrawerOpen = [drawer.drawerPosition isEqualToPosition:PulleyPosition.open];
     [self.cityVC setProperty:@(isDrawerOpen) forKey:@"isDrawerOpen"];
-    self.rnScrollView.scrollEnabled = isDrawerOpen;
+    self.scrollableTabView.userInteractionEnabled = isDrawerOpen;
     if (!isDrawerOpen) {
-        [self.rnScrollView setContentOffset:CGPointZero animated:YES];
+        UIScrollView *cityScrollView = FindCityScrollView(self.cityVC.view);
```

Comment:
Note that this change is _closer_ to what we'd like (all the vertical scroll views scroll to the top) but currently grabs whichever one we find first (the AllEvents one). Since it has the title, I figured this makes sense for now but I'll open a ticket to follow-up.

Reply:
https://artsyproduct.atlassian.net/browse/LD-451